charts/coturn/values.yaml

```
@@ -0,0 +1,26 @@
+# The amount of coturn instances to run.  NOTE: Only one coturn can run per node due
+# to `hostNetwork`. If this number is higher than the amount of nodes that can
+# be used for scheduling (Also see `nodeSelector`) pods will remain in a
+# pending state untill you add more capacity.
+replicaCount: 1
+
+image:
+  repository: quay.io/wire/coturn
+  pullPolicy: IfNotPresent
+  # overwrite the tag here, otherwise `appVersion` of the chart will be used
+  tag: ""
```

```suggestion
  # `appVersion` of the chart will be used as tag
```

Could you remove the tag here entirely, and always use the appVersion in Chart.yaml?

---

Is it possible to override `appVersion` at deployment time? My understanding is that exposing this handle for overriding the container image tag is useful for deploying different versions of images to test environments (e.g. testing a new version of coturn before burning the version number into the Helm chart which gets shipped to the public).